packages/gestalt/src/ExpandableModuleBase.js

```diff
@@ -0,0 +1,108 @@
+// @flow strict
+import React, { type Node } from 'react';
+import Box from './Box.js';
+import IconButton from './IconButton.js';
+import Icon from './Icon.js';
+import TapArea from './TapArea.js';
+import Text from './Text.js';
+
+type Props = {|
+  title: string,
+  icon?: string,
+  iconAccessibilityLabel?: string,
+  summary?: Array<string>,
+  isCollapsed: boolean,
+  onModuleClicked?: boolean => void,
+  type?: string,
+  children?: Node,
+|};
+
+const EXPANDABLE_TYPE_ATTRIBUTES = {
+  info: {
+    color: 'darkGray',
+  },
+  error: {
+    icon: 'workflow-status-problem',
+    color: 'red',
+    accessibilityLabel: 'Error icon',
+  },
+};
+
+export default function ExpandableModuleBase({
+  title,
+  icon,
+  iconAccessibilityLabel,
+  summary,
+  isCollapsed,
+  onModuleClicked,
+  type = 'info',
+  children,
+}: Props): Node {
+  const handleModuleClick = () => {
+    if (onModuleClicked) {
+      onModuleClicked(!isCollapsed);
+    }
+  };
+  return (
+    <>
+      <TapArea onTap={handleModuleClick}>
```

Comment:
@galaxy0101 didn't we say it would only open when you click on the arrow down/up?

Reply:
we said to expand it when click on the entire bar, not the entire block, particularly not the entire children area in TDD.

Reply:
@galaxy0101 https://react-accessible-accordion.springload.co.nz/ - let's make sure we add the correct `aria-` roles (expanded)

Reply:
1) Our expandable looks like the one in
Collapsing the last expanded item
If you set allowZeroExpanded to true then a solitary expanded item may be collapsed again.
2) will add accessibility_ExpandableModule_spec.js

Reply:
re request on clarification `on let's make sure we add the correct aria- roles (expanded) `https://www.loom.com/share/c98f07484ce84eb0830b3cb40fcf33d9